modules/ROOT/pages/latest-overview-concept.adoc

```diff
@@ -1,68 +1,177 @@
+
+For details, see xref:defining-sla-tiers.adoc[Reviewing SLA Tiers Concepts].
+
+=== Client Providers (Identity Providers)
 
-// talk about naming, instances
+You use client providers to enforce security and regulations in your business organization. Client providers authorize client applications. 
 
-After configuring the API, you can perform API management tasks, applying policies and setting up SLA tiers, assigning permissions to the API environment,
-versioning, viewing API analytics, and more.
+For details, see xref:configure-multiple-credential-providers.adoc[Configure Multiple Client Providers for Client Management].
 
-The API dashboard shows key information about a deployed API instance:
+=== Contracts
+
+Contracts are created between client applications and API instances. A client application requests access in Exchange. Either the owner of the API instance approves the request in API Manager, or the approval is automatic, depending on configuration. When the request is approved, the client application is registered.
+
+Contracts are enforced with either of the following:
+
+* SLA enforcement policies
+* Client enforcement policies
+
+For details, see xref:api-contracts-landing-page.adoc[Client Applications and Contracts].
+
+=== API Alerts
+
+An API alert (different from a Runtime Manager alert) is an alarm that flags one of the following:
+
+* The API request violates a policy.
+* Requests received by the API exceed a given number within a specified period of time.
+* The API returns a specified HTTP error code.
+* The API response time exceeds a specified timeout value.
+
+API Manager triggers alerts when states change from desirable to undesirable or vice versa. When an alert is triggered, API Manager sends an email notification to you and to anyone else you specify in the configuration.
+
+For details, see xref:using-api-alerts.adoc[Alerts].
+
+=== API Assets
+
+API assets, published by Exchange, are the components that comprise applications. Components include any of the following:
+
+* APIs (OAS, RAML, RAML fragments, HTTP, WSDL, and custom)
+* API groups
+* Policies
+* Examples
+* Templates
+* Modules
+* Connectors
+
+For details, see xref:exchange::about-sharing-assets.adoc[Share Assets].
+
+=== API Groups
+
+An API group is an API asset that enables organizations to publish a group of API instances as a single unit, so that applications can access the APIs as a group using one client ID. API groups are created in API Manager and published to Exchange.
+
+For details, see xref:api-groups-landing-page.adoc[API Groups].
+
+=== API Instances
+
+An API instance can exist in multiple environments, and you can create multiple instances of an API in one environment.
```

Comment:
Not sure if this is totally clear. Each API instance belongs to one environment, but you can have different API instances of an API on different environments.

Reply:
Fixed, thanks!